src/products/services/products.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+
+import { ProductEntity } from '@products/entities/product.entity';
+import { GetAllProductsException } from '@products/exceptions/get-all-products.exception';
+import { ProductsRepository } from '@products/repositories/products.repository';
+
+@Injectable()
+export class ProductsService {
+  private readonly logger: Logger = new Logger(ProductsService.name);
+
+  constructor(
+    @InjectRepository(ProductsRepository)
+    private readonly productRepository: ProductsRepository,
+  ) {}
+
+  async getAll(): Promise<ProductEntity[]> {
```

Comment:
I would use just Product as the name of the class.

Reply:
Well, yes, I can make this change

Reply:
Done!

Reply:
good job!